Makefile

```diff
@@ -1,5 +1,6 @@
 # Package configuration
 PROJECT = datasets
+COMMANDS = datasets/PublicGitArchive/borges-indexer/cmd/borges-indexer datasets/PublicGitArchive/pga
```

Comment:
multitool is still needed to generate the dataset

Reply:
good catch, added